entrypoint.sh

```diff
@@ -2,9 +2,9 @@
 
 AWS_REGION="${AWS_REGION:-us-east-1}"
 
-if [ -f ".app.json" ] && [ ! "${VAULT_TOKEN}" ]
+if [ "${ENCRYPTED_VAULT_TOKEN}" ] && [ ! "${VAULT_TOKEN}" ]
```

Comment:
yes, currently for prod. the KMS keys are different for both envs, so they won't decrypt properly